lambda.tf

```diff
@@ -104,5 +104,6 @@ resource "aws_s3_bucket_notification" "bucket_notification" {
     filter_prefix       = "ggcanary/AWSLogs/"
     filter_suffix       = ".json.gz"
   }
+  depends_on = [module.lambda_function]
```

Comment:
Note: I'm pretty sure that's not necessary, since we directly reference  `module.lambda` in this block (and the link is also shown in the output of `terraform graph`), but let's keep it just in case.